src/mjolnir/transitbuilder.cc

```diff
@@ -296,8 +296,6 @@ void ConnectToGraph(GraphTileBuilder& tilebuilder_local,
     // Add the node and directed edges
     nb.set_edge_index(edge_index);
 
-    // reset the access to defaults.
-    nb.set_access((kPedestrianAccess | kWheelchairAccess | kBicycleAccess));
```

Comment:
Keep the access mask inherited from the GTFS.

This should not break the multimodal costing which doesn't evaluate the node access, nor normal pedestrian routes since those nodes are connections between the transit graph and the pedestrian graph.